src/librustdoc/html/render/mod.rs

```diff
@@ -893,8 +895,14 @@ fn render_assoc_item(
             }
         };
         let vis = meth.visibility.print_with_space(meth.def_id, cx).to_string();
-        let constness =
-            print_constness_with_space(&header.constness, meth.const_stability(cx.tcx()));
+        // FIXME: Once https://github.com/rust-lang/rust/issues/67792 is merged, we can remove
+        // this condition.
```

Comment:
"merged" is confusing for issues, I think "closed" or "implemented" would be more clear.

Reply:
Let's go for implemented then.